src/modules/previewpane/README.md

```diff
@@ -102,11 +102,93 @@ FileExplorerPreviewSettings(
+
+```xml
+<Component Id="Module_PowerPreview" Guid="FF1700D5-1B07-4E07-9A62-4D206645EEA9" Win64="yes">
+        <!-- Files to include dll's for new Previewer and it's dependencies -->
+        <File Source="$(var.BinX64Dir)\modules\XYZPreviewer.dll" />
+        <File Source="$(var.BinX64Dir)\modules\Dependency.dll" />
+      </Component>
+      <Component Id="Module_PowerPreview_PerUserRegistry" Guid="CD90ADC0-7CD5-4A62-B0AF-23545C1E6DD3" Win64="yes">
+        <!-- Added a separate component for Per-User registry changes -->
+        <!-- Registry Key for Class Registration of new Preview Handler -->
+        <RegistryKey Root="HKCU" Key="Software\Classes\CLSID\{ddee2b8a-6807-48a6-bb20-2338174ff779}">
+          <RegistryValue Type="string" Value="XYZPreviewHandler.XYZPreviewHandler" />
+          <RegistryValue Type="string" Name="DisplayName" Value="XYZ Preview Handler" />
+          <RegistryValue Type="string" Name="AppID" Value="{CF142243-F059-45AF-8842-DBBE9783DB14}" />
+          <RegistryValue Type="string" Key="Implemented Categories\{62C8FE65-4EBB-45e7-B440-6E39B2CDBF29}" Value=""/>
+          <RegistryValue Type="string" Key="InprocServer32" Value="mscoree.dll" />
+          <RegistryValue Type="string" Key="InprocServer32" Name="Assembly" Value="SvgPreviewHandler, Version=$(var.Version).0, Culture=neutral" />
+          <RegistryValue Type="string" Key="InprocServer32" Name="Class" Value="XYZPreviewHandler.XYZPreviewHandler" />
+          <RegistryValue Type="string" Key="InprocServer32" Name="RuntimeVersion" Value="v4.0.30319" />
+          <RegistryValue Type="string" Key="InprocServer32" Name="ThreadingModel" Value="Both" />
+          <RegistryValue Type="string" Key="InprocServer32" Name="CodeBase" Value="file:///[ModulesInstallFolder]XYZPreviewHandler.dll" />
+          <RegistryValue Type="string" Key="InprocServer32\$(var.Version).0" Name="Assembly" Value="XYZPreviewHandler, Version=$(var.Version).0, Culture=neutral" />
+          <RegistryValue Type="string" Key="InprocServer32\$(var.Version).0" Name="Class" Value="XYZPreviewHandler.XYZPreviewHandler" />
+          <RegistryValue Type="string" Key="InprocServer32\$(var.Version).0" Name="RuntimeVersion" Value="v4.0.30319" />
+          <RegistryValue Type="string" Key="InprocServer32\$(var.Version).0" Name="CodeBase" Value="file:///[ModulesInstallFolder]XYZPreviewer.dll" />
+        </RegistryKey>
+        <!-- Add new previewer to preview handlers list -->
+        <RegistryKey Root="HKCU" Key="Software\Microsoft\Windows\CurrentVersion\PreviewHandlers">
+          <RegistryValue Type="string" Name="{Clsid-Guid}" Value="Name of the Previewer" />
+        </RegistryKey>
+        <!-- Add file type association for the new Previewer -->
+        <RegistryKey Root="HKCU" Key="Software\Classes\.xyz\shellex">
+          <RegistryValue Type="string" Key="{8895b1c6-b41f-4c1c-a562-0d564250836f}" Value="{Clsid-Guid}" />
+        </RegistryKey>
+      </Component>
+```
+
+### MSIX
```

Comment:
We should have a warning about this not being recommended, or remove it all together.

Reply:
Added a warning in `MSIX` section.